src/clients/pages/Signup/SignupForm/index.tsx

```diff
@@ -0,0 +1,148 @@
+"use client";
+import Link from "next/link";
+import Button from "~/clients/components/Button";
+import { useForm } from "@mantine/form";
+import {
+  Paper,
+  TextInput,
+  Box,
+  PasswordInput,
+  Text,
+  Title,
+  rem,
+  createStyles,
+} from "@mantine/core";
+
+const useStyles = createStyles((theme) => ({
+  frame1: {
+    height: "75vh",
+    width: "50vw",
+    maxWidth: "35rem",
+    borderRadius: "2.5rem",
+    position: "relative",
+    border: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.blue[1] : "black"
+    }`,
+    [theme.fn.smallerThan("sm")]: {
+      width: "80vw",
+    },
+  },
+  frame2: {
+    width: "100%",
+    maxWidth: "35rem",
+    display: "flex",
+    flexDirection: "column",
+    borderRadius: "2.5rem",
+    border: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.blue[1] : "black"
+    }`,
+    position: "absolute",
+    top: "8px",
+    right: "9px",
+    height: "100%",
+    [theme.fn.smallerThan("sm")]: {
+      width: "80vw",
+    },
+  },
+  title: {
+    color: theme.colorScheme === "dark" ? theme.white : theme.black,
+  },
+
+  form: {
+    paddingTop: rem(80),
+    backgroundColor: "transparent",
+    width: "80%",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "start",
+    justifyContent: "center",
+    margin: "3% auto",
+
+    [theme.fn.smallerThan("sm")]: {
+      maxWidth: "100%",
+    },
+  },
+}));
+
```

Comment:
@SachiGoto I meant to separate `useStyle()` and each component into different files and export/import `useStyle()`. Then we can share the same stylings with login/signup

Reply:
```
styles.ts 


import {
  createStyles,
  rem,
} from "@mantine/core";

export const formStyles = createStyles((theme) => ({
  frame1: {
    height: "75vh",
    width: "50vw",
    maxWidth: "35rem",
    borderRadius: "2.5rem",
    position: "relative",
    border: `1px solid ${
      theme.colorScheme === "dark" ? theme.colors.blue[1] : "black"
    }`,
    [theme.fn.smallerThan("sm")]: {
      width: "80vw",
    },
  },
  frame2: {
    width: "100%",
    maxWidth: "35rem",
    display: "flex",
    flexDirection: "column",
    borderRadius: "2.5rem",
    border: `1px solid ${
      theme.colorScheme === "dark" ? theme.colors.blue[1] : "black"
    }`,
    position: "absolute",
    top: "8px",
    right: "9px",
    height: "100%",
    [theme.fn.smallerThan("sm")]: {
      width: "80vw",
    },
  },
  title: {
    color: theme.colorScheme === "dark" ? theme.white : theme.black,
  },

  form: {
    paddingTop: rem(80),
    backgroundColor: "transparent",
    width: "80%",
    display: "flex",
    flexDirection: "column",
    alignItems: "start",
    justifyContent: "center",
    margin: "3% auto",

    [theme.fn.smallerThan("sm")]: {
      maxWidth: "100%",
    },
  },
}));




```

```
LoginForm.tsx 

import {formStyles} from "~/clients/pages/style";
<div className={formStyles.frame1}></div>
```

So I created style.ts and moved createStyle css there and called it formStyles. 
I was trying and see if I can import it and use some of the styles but I got an error... 
What am I doing wrong... ?? 

<img width="787" alt="Screenshot 2023-07-07 at 5 11 18 PM" src="https://github.com/van-squad/map-t3/assets/70562492/5a0602de-f1b7-43f1-85ea-b41e82b10b73">

Reply:
it seems like you forgot to put `const {classes} = formStyles()`
And others should be remain the same.

Reply:
Ahhhhhh you are so right.  Now it works! Thank you so much Yuko! Also, how come its ts file not tsx file?

Reply:
because it's just a function, not a react component